### ISSUE TITLE
Feature Request - Properly Display antenna signal strength 

### ISSUE THREAD

**Issue by Akunert2009** [hd-zero/hdzero-goggle#164]
![Screenshot (6)](https://user-images.githubusercontent.com/124201935/218234340-52dcfdec-e50a-4750-a5d5-155d38d63d6d.png)
Can the antenna signal strength be amended to display VLQ8 properly?  See attached screenshots.

Thanks!

**Comment by Zuldan** [comment]
Yep 100% need some sort of calibration like you do with Rapidfire and Fusion.

**Comment by FPVRishi** [comment]
Could this also be due to an SMA issue etc - some posted on facebook that the trace had lifted etc, resulting in poor signal being received and shown on one of the antennas. This was remedied by them reflowing the joint using a bit of wire to a suitable point on the board... https://www.facebook.com/groups/hdzero/permalink/572197678141531/ and https://www.facebook.com/groups/hdzero/permalink/571644371530195/

**Comment by Akunert2009** [comment]
> Could this also be due to an SMA issue etc - some posted on facebook that the trace had lifted etc, resulting in poor signal being received and shown on one of the antennas. This was remedied by them reflowing the joint using a bit of wire to a suitable point on the board... https://www.facebook.com/groups/hdzero/permalink/572197678141531/ and https://www.facebook.com/groups/hdzero/permalink/571644371530195/

I asked about this issue in the HDZero discord and it was indicated that Carl himself has experienced this and said it was a calibration issue.  I do not know if it is related to lifted traces.  Perhaps someone else has additional insight.

**Comment by Zuldan** [comment]
I’m pretty sure it’s a calibration issue. 

It would be great to have something like the following in the Goggles menu.

Main Menu -> Tools 

RF Calibration
Focus Chart

**Comment by cynfewl** [comment]
Does this issue primarily present as lower signal strength on the left two antenna icons? I have this event at very close range and am wondering if it's this calibration issue or physical damage. I don't get zero bars, but the left 2 icons are generally about half the ones on the right.

Is there information somewhere on which antenna the icons refer to?

**Comment by Zuldan** [comment]
My left two antenna icons are also usually lower than the right two but only by 1 bar. Exactly what is shown in the original post above. I remember seeing somewhere that the HDZero engineers have confirmed it's not a hardware issue, but a firmware issue. 

There are very very rare cases where the antenna adapter hasn't been soldered on correctly but I think in that case you would see 1 bar to nothing, all the time.
